dataframes/03-from-pandas-to-dask.ipynb

```diff
@@ -0,0 +1,891 @@
+    "For more detials see [issue#653 on github](https://github.com/dask/dask/issues/653)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1 Rename"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pandas\n",
+    "print(pdf.columns)\n",
+    "pdf.rename(columns={'id':'ID','name':'Name','x':'coor_x', 'y':'coor_y'},inplace=True)\n",
+    "pdf.columns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dask\n",
+    "# Must update using the correct sequence of the columns\n",
+    "print(ddf.columns)\n",
+    "ddf.columns = ['ID','Name','coor_x','coor_y']\n",
+    "ddf.columns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.2 Column munipilations  \n",
+    "There are several diffrences when manimuplating data. mose of which require to ran a couple of lines (instead of one-liners)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Convert index into Time column"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pandas\n",
+    "pdf = pdf.assign(Time=pd.to_datetime(pdf.index).time)\n",
+    "pdf.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dask\n",
+    "ddf = ddf.assign(Time=ddf.index)\n",
+    "ddf['Time'] = ddf['Time'].dt.time\n",
+    "ddf.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.3 Drop NA on column"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pandas\n",
+    "pdf = pdf.assign(colna = None)\n",
+    "print(pdf.head())\n",
+    "pdf.dropna(axis=1, how='all', inplace=True)\n",
+    "print(pdf.head())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dask\n",
+    "ddf = ddf.assign(colna = None)\n",
+    "print(ddf.head())\n",
+    "if ddf.colna.isnull().all().compute() == True:   # check if all values in column are Null - VERY slow\n",
+    "    ddf = ddf.drop(labels=['colna'],axis=1)\n",
+    "print(ddf.head())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##  1.4 Reset Index"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pandas\n",
+    "pdf.reset_index(drop=True, inplace=True)\n",
+    "pdf.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Dask is in a development mode\n",
```

Comment:
I don't think this is very useful. It will go out of date when the bug is fixed (it actually seems to be fixed already).

Reply:
Great!
I'll update the cell without the workaround